lib/internal/crypto/cipher.js

```diff
@@ -38,19 +43,25 @@ const { deprecate, normalizeEncoding } = require('internal/util');
 // Lazy loaded for startup performance.
 let StringDecoder;
 
-function rsaFunctionFor(method, defaultPadding) {
+function rsaFunctionFor(method, defaultPadding, keyType) {
   return function(options, buffer) {
-    const key = options.key || options;
+    const { format, type, data, passphrase } =
+      keyType === 'private' ?
```

Comment:
could be misreading, but does this mean a `keyType = 'looney-tunes'` is accepted by the API, and means "not private"? Might be worth making sure only reasonable strings are provided.

Reply:
This is a private parameter, it cannot be passed by users. It is only used to construct `publicEncrypt`, `privateDecrypt`, `privateEncrypt` and `publicDecrypt`.